components/slices/LatestNews.vue

```diff
@@ -75,8 +75,15 @@ export default {
   margin-bottom: 118px;
 
   &__list {
+    margin: 0 auto;
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+
+    @media (min-width: 700px) {
```

Comment:
**Question :**  N'envisagerait-on pas dans le futur d'inclure cette taille aux formats déjà existants afin d'éviter de combiner `@include device-is` et media queries ?